src/config.tsx

```diff
@@ -1 +1,2 @@
 export const BACKEND_URL = 'http://localhost:8080/api';
+export const BACKEND_URL_BASE = 'http://localhost:8080';
```

Comment:
```suggestion
export const BACKEND_URL_BASE = 'http://localhost:8080';
export const BACKEND_URL = '${BACKEND_URL_BASE}/api';
```
You can combine these 2 urls